Assets/Resources/JSON/Languages.json

```diff
@@ -0,0 +1,243 @@
+{
+  "Lookup" : [
```

Comment:
The spreadsheet is broken down by language and then uses some formulas to compile everything onto a single sheet: https://docs.google.com/spreadsheets/d/1xxKYxQnxLpgyFR0JCh7TLKkQLqFm7c_rXV1G7FDHWto/edit#gid=1409916858. Let me know what you think of the format.

A single JSON is easier to parse for our purposes, so I remain in favor of it.

As for words without direct translations, definitely a trickier question. Seems like the only true solution is NLP (natural language processing). But perhaps we can think on this more down the line